descriptor.md

```diff
@@ -19,7 +19,8 @@ The following fields contain the primary properties that constitute a Descriptor
 
 - **`mediaType`** *string*
 
-  This REQUIRED property contains the MIME type of the referenced content.
+  This REQUIRED property contains the media type of the referenced content.
+  Values MUST comply with [RFC 6838][rfc6838].
```

Comment:
> Most of the rest of the specification is related to registration

Doesn't all the registration stuff still apply?  For example [§3.2][1] has:

> While public exposure and review of media types to be registered in the vendor tree are not required, using the media-types@iana.org mailing list for review is encouraged, to improve the quality of those specifications.

so I think our unregistered vendor types are fine.  And I don't think our spec should allow unregistered standards-tree type (violating [§3.1]) or other RFC-violating media types, even if they happen to match the §4.2 syntax.

If you'd like to draw attention to §4.2 in particular, I'd be fine with:

> Values MUST comply with [RFC 6838][2], including the naming requirements in it's [§4.2][3].

[1]: https://tools.ietf.org/html/rfc6838#section-3.2
[2]: https://tools.ietf.org/html/rfc6838
[3]: https://tools.ietf.org/html/rfc6838#section-4.2
[4]: https://tools.ietf.org/html/rfc6838#section-3.1